### PR TITLE
Adding README for deploying flux and demo chart

### PR DESCRIPTION
**Pull request by efiacor**
Start on the README for deploying flux and demo pkg in Nephio

### CODE REVIEW

**Review by lapentad — APPROVED** [review]

**Review by liamfallon — APPROVED** [review]

File: demo/README.md [hunk 1]
```diff
@@ -0,0 +1,139 @@
+# Deploying helm chart in Nephio using Flux CD
+
+[Flux CD](https://fluxcd.io/flux/use-cases/helm/) provides a set of kubernetes controllers to enable a GitOps driven deployment of helm charts, using relevant the flux Custom Resources.
+
+In this example, we deploy the flux [helm](https://fluxcd.io/flux/components/helm/) and [source](https://fluxcd.io/flux/components/source/) controllers via a kpt package to the target workload cluster.
+
+Then, we can utilize the flux Custom Resources defined in another test kpt package to deploy an example helm chart.
+
+##  Pre requisites:
```
Comment (liamfallon) [inline demo/README.md:9]: All one word "Prerequisites:?

**Review by liamfallon — COMMENTED** [review]
Would it be worthwhile to add a statement saying that the online boutique kpt package on Nordix points at the official online boutique helm repo/charts.

File: demo/README.md (hunk 1 above)
Comment (efiacor) [reply demo/README.md]: done

**Comment by efiacor** [comment]
> Would it be worthwhile to add a statement saying that the online boutique kpt package on Nordix points at the official online boutique helm repo/charts.

done

**Pull request merged**